mirdata/datasets/fma_keys.py

```diff
@@ -0,0 +1,280 @@
+        filename="000-019.zip",
+        url="https://zenodo.org/records/10719860/files/000-019.zip?download=1",
+        checksum="b86f6414820c1422b2c6cdf87be1ef3a",
+    ),
+    "tracks-020-039": download_utils.RemoteFileMetadata(
+        filename="020-039.zip",
+        url="https://zenodo.org/records/10719860/files/020-039.zip?download=1",
+        checksum="a2da8377fdbc1d3a1f54dd60aa7b8f9b",
+    ),
+    "tracks-040-049": download_utils.RemoteFileMetadata(
+        filename="040-049.zip",
+        url="https://zenodo.org/records/10719860/files/040-049.zip?download=1",
+        checksum="d70babe5f66bdf3e821c42a8b8aafb9b",
+    ),
+    "tracks-050-059": download_utils.RemoteFileMetadata(
+        filename="050-059.zip",
+        url="https://zenodo.org/records/10719860/files/050-059.zip?download=1",
+        checksum="f53fcba704fce27e5c7f3ec2532dcb44",
+    ),
+    "tracks-060-069": download_utils.RemoteFileMetadata(
+        filename="060-069.zip",
+        url="https://zenodo.org/records/10719860/files/060-069.zip?download=1",
+        checksum="1520f067d7caaf0813780ff69bc4ba85",
+    ),
+    "tracks-070-079": download_utils.RemoteFileMetadata(
+        filename="070-079.zip",
+        url="https://zenodo.org/records/10719860/files/070-079.zip?download=1",
+        checksum="186643746fcb1f4722a28d3eb9c6b99c",
+    ),
+    "tracks-080-089": download_utils.RemoteFileMetadata(
+        filename="080-089.zip",
+        url="https://zenodo.org/records/10719860/files/080-089.zip?download=1",
+        checksum="8cf882609fc2f301621c2e9f9da03214",
+    ),
+    "tracks-090-099": download_utils.RemoteFileMetadata(
+        filename="090-099.zip",
+        url="https://zenodo.org/records/10719860/files/090-099.zip?download=1",
+        checksum="84f0f036e3778ffd97c10b591f803d06",
+    ),
+    "tracks-100-109": download_utils.RemoteFileMetadata(
+        filename="100-109.zip",
+        url="https://zenodo.org/records/10719860/files/100-109.zip?download=1",
+        checksum="4a307f019d3354064814f05d1dffa1e2",
+    ),
+    "tracks-110-124": download_utils.RemoteFileMetadata(
+        filename="110-124.zip",
+        url="https://zenodo.org/records/10719860/files/110-124.zip?download=1",
+        checksum="88d7dbcca82189ed75b7baa5aa132fc1",
+    ),
+}
+
+KEY_MAP = {
+    "C": 0,
+    "C#": 1,
+    "D": 2,
+    "D#": 3,
+    "E": 4,
+    "F": 5,
+    "F#": 6,
+    "G": 7,
+    "G#": 8,
+    "A": 9,
+    "Bb": 10,
+    "B": 11,
+}
+
+MODE_MAP = {"minor": 0, "Major": 1}
+
+
+class Track(core.Track):
+    """FMA Keys Track class
+
+    Args:
+        track_id (str): track id of the track
+
+    Attributes:
+        spotify_uri (str): Spotify URI if available
+        key (str): path to the track's audio file
+        mode (str): path to the track's audio file
+        key_number (int): path to the track's audio file
+        mode_number (int): path to the track's audio file
+        audio (str): path to the track's audio file
```

Comment:
most of them seem to have `path to the track's audio file` as description. Please update! also please change `audio` for `audio_`path` .